conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
could you change filename to vsan instead of vmfs

Reply:
Done